src/preferences.py

```diff
@@ -82,6 +83,9 @@ class PreferencesWindow(Adw.PreferencesWindow):
     legendary_config_action_row = Gtk.Template.Child()
     legendary_config_file_chooser_button = Gtk.Template.Child()
 
+    retroarch_expander_row = Gtk.Template.Child()
```

Comment:
dont forget the action row to display the resolved location (or a warning icon if unresolvable)
```py
retroarch_config_action_row = Gtk.Template.Child()
```